apiserver/apiserver.go

```diff
@@ -664,3 +676,35 @@ func serverError(err error) error {
 	}
 	return nil
 }
+
+func (srv *Server) processModelRemovals() error {
+	w := srv.state.WatchModels()
+	defer w.Stop()
+	for {
+		select {
+		case <-srv.tomb.Dying():
+			return tomb.ErrDying
+		case modelIDs := <-w.Changes():
+			for _, modelID := range modelIDs {
```

Comment:
these are modelUUIDs, can we rename the vars to avoid the notion there maybe such a thing as a modelID